cmd/mailchain/internal/http/handlers/protocols.go

```diff
@@ -33,15 +33,25 @@ func GetProtocols(base *settings.Root) func(w http.ResponseWriter, r *http.Reque
 			continue
 		}
 
-		networks := []string{}
+		networks := []Network{}
 
 		for _, network := range protocol.Networks {
-			if !network.Disabled() {
-				networks = append(networks, network.Kind())
+			if !network.Disabled() && protocol.Kind == "ethereum" {
+				networks = append(networks, Network{Name: network.Kind(), ID: ""})
+			} else if !network.Disabled() && protocol.Kind == "substrate" {
```

Comment:
Can you create some unit tests for these branches https://coveralls.io/builds/27076165/source?filename=cmd/mailchain/internal/http/handlers/protocols.go